### DISCUSSION TITLE
Push for 0.2.0 release this week?

### DISCUSSION THREAD

**Discussion by cholmes** [opengeospatial/geoparquet#61] [Administrative]
It'd be great to get a 0.2.0 release out ahead of [cloud-native geo outreach event](https://github.com/opengeospatial/geoparquet) as we'll likely get a lot more people checking it out. 

We have a lot of PR's with solid improvements, that I think perhaps no one feels fully empowered to merge? 

It could be worthwhile to have a synchronous session with a few people online at once to finalize any discussion and move forward.

And/or Might be worth setting that 2 approvals from existing contributors and X days (3?) of time open (or 4+ reviewers?) for anyone to sound in is sufficient to merge.

**Comment by alasarr** [comment]
How about including https://github.com/opengeospatial/geoparquet/pull/60 at 0.2.0?

**Comment by cholmes** [comment]
Yeah, that one is essential to me. Or at least us resolving the discussion around it. Would be good to get a couple more people sounding in.
